Gtk-4.0.gir

```diff
@@ -73519,6 +74180,9 @@ when the widget has focus.</doc>
           </parameter>
         </parameters>
       </method>
+      <property name="item-type" transfer-ownership="none">
```

Comment:
Sorry I missed this one, but it lacks a since annotation upstream

Reply:
Fixed upstream https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4827